### PR TITLE
40510 add a feature flag to enable mocks

### PR DESCRIPTION
**Pull request by randomsync**
## Description of change

This PR adds a feature flag to enable/disable mocks for downstream services (LoROTA, CHIP)

## Original issue(s)
department-of-veterans-affairs/va.gov-team#40510

## Things to know about this PR

- feature flag: `check_in_experience_mock_enabled`

### CODE REVIEW

File: modules/check_in/app/services/v2/chip/client.rb [hunk 1]
```diff
@@ -185,6 +185,10 @@ def default_headers
           'x-apigw-api-id' => tmp_api_id
         }
       end
+
+      def mock_enabled?
+        settings.mock || Flipper.enabled?('check_in_experience_mock_enabled') || false
```
[the remark this answers is not in the document]
Comment (kanchanasuriya) [reply modules/check_in/app/services/v2/chip/client.rb:190]: Can I pls know what does `settings.mock` would do?

[the remark this answers is not in the document]
Comment (randomsync) [reply modules/check_in/app/services/v2/chip/client.rb:190]: it gets the setting called `mock` from the settings file (https://github.com/department-of-veterans-affairs/vets-api/blob/04e27e1c6118191991516e4d74325796625d79f9/modules/check_in/app/services/v2/lorota/client.rb#L51)

**Review by kanchanasuriya — DISMISSED** [review]
LGTM

**Comment by randomsync** [comment]
superceded by #9803

**Pull request closed without merging**